examples/nodejs/compression/configuration.ts

```diff
@@ -0,0 +1,4 @@
+configuration.withCompression({
+  compressionExtensions: CompressionExtensions.load(),
+  automaticDecompression: AutomaticDecompression.Enabled,
```

Comment:
Why is there an AutomaticDecompression ability in SDK but not a AutomaticCompression setting? Does this mean I always need to explicitly pass field to enable setring when setting value but not when reading?